src/components/picker/PickerItemsList.tsx

```diff
@@ -66,7 +67,14 @@ const PickerItemsList = (props: PickerItemsListProps) => {
   },
   [children]);
 
+  const renderPropItems = useCallback(({item}: ListRenderItemInfo<PickerItemProps>) => {
+    return <PickerItem value={item.value} label={item.label} disabled={item.disabled}/>;
```

Comment:
This can be simpler:
```suggestion
    return <PickerItem {...item}/>;
```

Reply:
And also, you're supporting `disabled` here but not in the type (and API)

Reply:
I think you missed this one